Libraries/ConnectNIO/Internal/GRPCInterceptor.swift

```diff
@@ -186,3 +187,18 @@ extension GRPCInterceptor: StreamInterceptor {
         }
     }
 }
+
+private final class Locked<T>: @unchecked Sendable {
```

Comment:
Re-implements `Locked` using NIO's locks so that the `Connect.Locked` can stay internal